### DISCUSSION TITLE
Livereload as other js file in the same port

### DISCUSSION THREAD

**Discussion by pitakill** [thgh/rollup-plugin-livereload#78] [Q&A]
Hi there!, great work you have here! :-)

I'm currently learning Svelte and trying to add it to my workflow.

My workflow is all my services (frontend and backend) are docker containers and I use `nginx-proxy` to serve my services with a valid ssl certificate issued by Let's Encrypt.
I noticed that the livereload plugin that use the oficial Svelte bootstrap use this plugin, digging more I noticed that use a different port to serve the `js` file that handles the reload when a file is changed. In my workflow this become a lose of the great functionality that when I change a file, the browser reloads and my changes are in the browser.

So my question is: Is there a way to serve the `js` file that handles the reload in the same port as my server is? (ie. localhost:5000 -> for my work, and localhost:35729 become localhost:5000 and localhost:5000/livereload.js)

**Comment by kipanshi** [comment]
I'm trying to solve the same problem, looking now if I can somehow use Requestly to overwrite the host,
also we're using https, so need to enable https in the plugin config.

**Comment by thgh** [comment]
I haven't seen a livereload server that runs on the same port. Perhaps you can proxy a second port with nginx? If you need an extra option `scriptOrigin: 'https://nginxproxy:33333'` then I will merge the PR.

**Comment by pitakill** [comment]
Sounds good! :-)

`create-react-app` use a file that is served in the same port that the project itself, so the integration with docker is flawless, that is how actually work with frontend development. But Svelte and the paradigm that uses is a deal breaker with React.

**Comment by rizary** [comment]
@pitakill do you have some fix in this matter? Not only svelte, but my current react project suffered from this issue. I use load balancer in my `docker-compose.yaml`, and I can't do development in there.

**Comment by jonkoops** [comment]
Since this is more of a question than a bug of feature request I am converting this to a discussion.

The stack used here is quite complex and involves several different technologies outside of this library, so I would recommend trying to create a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example). This makes it easier to discuss the issue at hand, eliminating any guesswork.
